cmd/builder/internal/builder/main_test.go

```diff
@@ -289,52 +395,62 @@ func makeModule(dir string, fileContents []byte) error {
 	return nil
 }
 
+func replaceModules() []string {
```

Comment:
Why func vs a regular variable/constant?

Reply:
String arrays can't be constants in Go, so I thought making it a function would prevent mutation. A variable is also fine though since this is just a test, I've updated it accordingly.